db/config.hh

```diff
@@ -384,6 +384,7 @@ public:
     const db::extensions& extensions() const;
 
     utils::UUID host_id;
+    utils::UUID instance_id;
```

Comment:
Why do we have both host_id and instance_id?

Reply:
Because host_id is not unique per instance.

Reply:
How come?

Reply:
https://github.com/scylladb/scylladb/issues/9839#issuecomment-1076061960

Reply:
Let's perform that prerequisite then. I don't want a node id, instance id, raft group 0 id to all live together and when I see some uuid to search three lists to find it.